debezium-core/src/main/java/io/debezium/metrics/Metrics.java

```diff
@@ -31,11 +32,11 @@
     private volatile boolean registered = false;
 
     protected Metrics(CdcSourceTaskContext taskContext, String contextName) {
-        this.name = metricName(taskContext.getConnectorType(), taskContext.getConnectorName(), contextName);
+        this.name = metricName(taskContext.getConnectorType(), taskContext.getConnectorName(), contextName, taskContext.getConnectorProperties());
```

Comment:
Hey @jpechane @shichao-an @gunnarmorling @Naros I have a generic question about this change.

https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/metrics/Metrics.java#L37
looks to me like we expect the connector name to be passed as name for the MBean, but instead most implementation seem to use the logical server name. Either the `CdcSourceTaskContext#getConnectorName()` method name is misleading or is it maybe an issue that connector implementations assign the logical server name, for example: https://github.com/debezium/debezium/blob/main/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresTaskContext.java#L44